openpaas-james/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/openpaas/james/common/LinagoraKeystoreSetMethodContract.scala

```diff
@@ -500,6 +500,224 @@ trait LinagoraKeystoreSetMethodContract {
          |}""".stripMargin)
   }
 
+  @Test
+  def keystoreSetDestroyShouldSucceed(): Unit = {
+    val request = s"""{
+                     |	"using": ["urn:ietf:params:jmap:core", "com:linagora:params:jmap:pgp"],
+                     |	"methodCalls": [
+                     |		["Keystore/set", {
+                     |			"accountId": "$ACCOUNT_ID",
+                     |			"create": {
+                     |        "K87": {
+                     |          "key": "$PGP_KEY_ARMORED"
+                     |        }
+                     |      }
+                     |		}, "c1"],
+                     |    ["Keystore/set", {
+                     |			"accountId": "$ACCOUNT_ID",
+                     |			"destroy": [
+                     |        "$PGP_KEY_ID"
+                     |      ]
+                     |		}, "c2"]
+                     |	]
+                     |}""".stripMargin
```

Comment:
Can we add a `Keystore/get` in the call? to ensure that the key have well been destroyed?